net-dns/bind/bind-9.14.4.ebuild

```diff
@@ -251,10 +254,11 @@ src_install() {
 	# bug 450406
 	dosym named.cache /var/bind/root.cache
 
-	dosym /var/bind/pri /etc/bind/pri
-	dosym /var/bind/sec /etc/bind/sec
-	dosym /var/bind/dyn /etc/bind/dyn
+	dosym "${ED%/}/var/bind/pri" "/etc/bind/pri"
```

Comment:
I am not sure that "linked to" objects must not respect prefix as well.

Reply:
This is working and also eliminates the current QA warnings.